.gitignore

```diff
@@ -464,6 +464,7 @@ project/metals.sbt
 # mdoc
 website/node_modules
 website/build
+website/yarn.lock
```

Comment:
Why ignore yarn.lock?

Reply:
this was just copied from the master branch  - I will remove